xbmc/cores/VideoPlayer/VideoPlayer.h

```diff
@@ -478,7 +480,7 @@ class CVideoPlayer : public IPlayer, public CThread, public IVideoPlayer,
   void UpdateContent();
   void UpdateContentState();
 
-  void UpdateFileItemStreamDetails(CFileItem& item);
+  void UpdateFileItemStreamDetails(CFileItem& item, bool alwaysUpdate = false);
```

Comment:
Another default and another bool parameter... I hope you have seen my recent comments regarding these anti-patterns?